lib/core/rancher/step.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# IH_CORE_DIR will be set to the directory containing the bin and lib directories.
+
+function ih::setup::core.rancher::help() {
+  echo 'Install Rancher Desktop as an alternative to Docker Desktop'
+}
+
+# Check if the step has been installed and return 0 if it has.
+# Otherwise return 1.
+function ih::setup::core.rancher::test() {
+
+    # Check if .rd directory exists as well as rdctl and docker command
+    if command -v rdctl  &&  command -v docker && [ -d "$HOME/.rd/bin" ]  ; then
+
+        # Check for symlinks
+        DOCKERBIN=/usr/local/bin/docker
+        DOCKERCOMPOSEBIN=/usr/local/bin/docker-compose
+        if [ -f "$DOCKERBIN" ] && [  -f "$DOCKERCOMPOSEBIN" ]; then
+            # Check if kubernetes is enabled
+            KUBERNETESENABLED=$(rdctl list-settings | jq  '.kubernetes.enabled')
+            if [ "$KUBERNETESENABLED" = "false" ]; then
+                return 0
+            fi
+        fi
+    fi 
+    
+    ih::log::debug "Rancher Desktop is not available"
+    return 1
+
+}
+
+function ih::setup::core.rancher::deps() {
+  echo "core.shell"
+}
+
+
+function ih::setup::core.rancher::install() {
+    CASKSUCCEEDED=1
+    # Installation and configuration of Rancher Desktop
+    for _ in 1 2 3; do
+
+        # Detect Rosetta
+        if [[ $(sysctl -n sysctl.proc_translated) -eq 1 ]]; then
+            # Rosetta  Active
+            arch -arm64 -c brew install ih-rancher
+        else
+            brew install ih-rancher
+        fi
+
+        CASKSUCCEEDED=$?
+        if [ $CASKSUCCEEDED -eq 0 ]; then
+            # Disable kubernetes by default. 
+            # More information: https://docs.rancherdesktop.io/references/rdctl-command-reference/#rdctl-set
+            # Rancher Desktop has to be running in order to disable Kubernetes
+            rdctl start --container-engine moby &
+            echo "Starting Rancher"
+            while :
+            do	
+                sleep 5
+                rdctl set --kubernetes-enabled=false > /dev/null 2>&1
```

Comment:
I'm wondering if we really need to wait for Rancher Desktop to be up in order to do this. The config file is located at `$HOME/Library/Preferences/rancher-desktop/settings.json`, we can probably just use `jq` to update it in place. We can update `core.rancher::test` to also use `jq` directly against that file, which has the nice effect of also just failing if that file doesn't exist.

Working with the file directly is probably more fragile than using the CLI, since the config file structure isn't documented, but since we control the version of Rancher Desktop, we can re-test this before we upgrade.